File: codewars/7kyu/string-ends-with/dduck-string-ends-with.py
# https://www.codewars.com/kata/51f2d1cafc9c0f745c00037d

from unittest import TestCase

solution = lambda str, ending: str.endswith(ending)

TestCase().assertEqual(solution('abcde', 'cde'), True)
TestCase().assertEqual(solution('abcde', 'abc'), False)
TestCase().assertEqual(solution('abcde', ''), True)

File: codewars/7kyu/string-ends-with/dduck-string-ends-with.js
// https://www.codewars.com/kata/51f2d1cafc9c0f745c00037d

const solution = (str, ending) => str.endsWith(ending);

console.log(solution('abcde', 'cde'), true)
console.log(solution('abcde', 'abc'), false)
